jschon/vocabulary/annotation.py

```diff
@@ -3,6 +3,7 @@
 from jschon.vocabulary import AnnotationKeyword
 
 __all__ = [
+    'AnnotationKeyword',
```

Comment:
Sorry, I meant add it to `__all__` in `vocabulary/__init__.py`. It should be exported from the module where it's defined.

`__all__` is not only relevant to `import *`, perhaps more importantly it represents the public API of a module. I picked this up because my IDE highlights imports of non-public module members:

![__all__](https://user-images.githubusercontent.com/52427991/223030644-726cf580-e241-4bf9-9041-0d6797ab9c4e.png)